libc/src/stdio/generic/CMakeLists.txt

```diff
@@ -319,7 +319,7 @@ add_entrypoint_object(
 add_entrypoint_object(
   getchar_unlocked
   SRCS
-    getc_unlocked.cpp
+    getchar_unlocked.cpp
   HDRS
     ../getc_unlocked.h
```

Comment:
the header needs to be fixed as well.

Reply:
Missed that, probably still ended up compiling since it just used an implicit definition of the function. I wonder if there's a way to prevent that statically somehow.

Reply:
Shouldn't you be able to pipe a file via CMake or something?

Reply:
probably? It's just not something I've looked into that closely. The functions that take input from stdin all have corresponding functions that take input from a provided file, so we generally just test with those. Of course, this is revealing that we really should have tests for the stdin functions, even if they're just smoke tests.

Reply:
fixed in f51ea4b9e009f449cd4008e4e62eaa43e77ec10d PTAL